docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/amazon-eks.md

```diff
@@ -31,6 +31,43 @@ that uses the Amazon EBS `gp3` volume type. Then, use it cluster-wide as a defau
 If you encounter issues with EBS CSI Driver, follow the instructions in the [helm-profiles repository](https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/main/aws/README.md#ebs-csi-driver-addon) maintained by the Camunda Consulting Team.
 :::
 
+## Load Balancer Setup
+
+AWS is offering different types of Load Balancers (LB). Those namely being Classic Load Balancer (CLB), Network Load Balancer (NLB), and Application Load Balancer (ALB). Typically the NLB and ALB are used in production setups.
+
+The zeebe-gateway requires gRPC to work, which in itself requires http2 to be used. Additionally, it's recommended to secure the endpoint with TLS.
+
+Here the choice of Load Balancer is important as not every setup will work with every TLS termination. Typically, the NLB has to terminate the TLS within the ingress, while the ALB can terminate TLS within the Load Balancer, allowing the usage of the [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/).
+
+The NLB will not work with the AWS Certificate Manager, as the ACM does not allow exporting the private key required to terminate the TLS within the ingress.
+
+The C8 helm chart primarily focuses on the [ingress-nginx controller](https://github.com/kubernetes/ingress-nginx), due to the usage of controller specific annotations. Using a different ingress controller will require supplying the necessary equivalent annotation options as well as making sure that http2 is enabled and gRPC is used for the zeebe-gateway.
+
+To conclude for using the **Application Load Balancer** (ALB), one requires:
```

Comment:
I'd make this into a level `###` heading which then also allows us in the future to provide customers a deep link to this section about ALBs on the docs page, same for the NLB below. I think the topic will come up again and having a URL ready is useful to help people.